dataland-document-manager/src/main/kotlin/org/dataland/documentmanager/DatalandDocumentManager.kt

```diff
@@ -1,15 +1,21 @@
 package org.dataland.documentmanager
 
+import org.dataland.datalandbackendutils.email.EmailSender
 import org.dataland.documentmanager.configurations.OpenAPIConfiguration
 import org.springframework.boot.autoconfigure.SpringBootApplication
 import org.springframework.boot.runApplication
 import org.springframework.context.annotation.ComponentScan
+import org.springframework.context.annotation.ComponentScan.Filter
+import org.springframework.context.annotation.FilterType
 
 /**
  * Class to define the spring boot application
  */
 @SpringBootApplication
-@ComponentScan(basePackages = ["org.dataland"])
+@ComponentScan(
+    basePackages = ["org.dataland"],
+    excludeFilters = [Filter(type = FilterType.ASSIGNABLE_TYPE, classes = [EmailSender::class])],
```

Comment:
Same as the others

Reply:
Same as the others

Reply:
i will close them and leave the original one open